.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ Fixes # (issue)
 ## Type of change
 
 Please add a line in the relevant section of
-[CHANGELOG.md](https://github.com/SGIModel/StarMuse/blob/development/CHANGELOG.md) to
+[CHANGELOG.md](https://github.com/EnergySystemsModellingLab/StarMuse/blob/development/CHANGELOG.md) to
```

Comment:
```suggestion
[CHANGELOG.md](https://github.com/EnergySystemsModellingLab/MUSE_OS/blob/development/CHANGELOG.md) to
```

A few other references to `StarMuse` which may also need to be changed. (Also I'm not sure this comment about the changelog is still relevant, but I guess that's a separate issue)

Reply:
Actually I think all the other references to `StarMuse` are correct, but this one needs to be changed

Reply:
Good spot! Thank you for reading through this more carefully than I did :laughing:

Reply:
Not that we actually use the changelog anymore, but it might as well be correct!